src/Traits/HasRoles.php

```diff
@@ -314,6 +314,20 @@ public function getRoleNames(): Collection
         return $this->roles->pluck('name');
     }
 
+    /** Return only single rolename, developer don't need to use an arr key always.
+     * @param integer $key always sets to 0.
+     * @return String $role_name
+     */
+    public function getRoleName($key = 0): String
+    {
+        if (FALSE === is_int($key)) {
+            trigger_error('function getRoleName expected Argument 1 to be Integer', E_USER_WARNING);
+        }
+        if (is_int($key)) {
+            return isset($this->roles[$key]->name) ? $this->roles[$key]->name : '';
```

Comment:
> isset($this->roles[$key]->name)
Don't seems to be right

Reply:
can you please review it again @fredsal 
also, can you please guide me @fredsal to improve/fix this function.